<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="show_answer">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>120</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Answer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="next_question">
   <property name="geometry">
    <rect>
     <x>144</x>
     <y>470</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Next Question</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="answerText">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>481</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="questionText">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>481</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>160</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Answer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="end_quiz">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>470</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>End Quiz</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Question</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="correctRadio">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>420</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Correct</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="incorrectRadio">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>420</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Incorrect</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
